### PR TITLE
Distinguish master and standby master when installing PXF

### PR DESCRIPTION
**Pull request by oliverralbertini**
Authored-by: Oliver Albertini <oalbertini@pivotal.io>

### CODE REVIEW

**Comment by oliverralbertini** [comment]
This should only needs to be merged in when https://github.com/pivotal/gp-concourse-cluster-provisioner/pull/63 is merged.

The above PR will allow us to use CCP to standup an env with standby master called `smdw`.

**Comment by frankgh** [comment]
@oliverralbertini this seems to be part of this PR as well: https://github.com/greenplum-db/pxf/pull/147

**Comment by oliverralbertini** [comment]
Yeah, it is needed to get the standby master successfully on the GPDB cluster for the pipeline. It won't be merged as part of the other PR.

**Comment by oliverralbertini** [comment]
OK, it's time to merge this, since I have merged the PR for CCP: https://github.com/pivotal/gp-concourse-cluster-provisioner/pull/63

File: concourse/scripts/install_pxf.bash [hunk 1]
```diff
@@ -4,7 +4,7 @@ set -euxo pipefail
 
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 GPHOME="/usr/local/greenplum-db-devel"
-MASTER_HOSTNAME=$( < cluster_env_files/etc_hostfile grep "mdw.*" | awk '{print $2}')
+MASTER_HOSTNAME=$(grep -E '\bmdw.*' cluster_env_files/etc_hostfile | awk '{print $2}')
```
Comment (frankgh) [inline concourse/scripts/install_pxf.bash]: what is the hostname of the standby master?

**Comment by frankgh** [comment]
are there any other scripts that use the master hostname in a similar way?

File: concourse/scripts/install_pxf.bash (hunk 1 above)
Comment (oliverralbertini) [reply concourse/scripts/install_pxf.bash]: standby master is called `smdw` (and master is just `mdw`). So without this fix, the variable `MASTER_HOSTNAME` gets set to something like:
```
'mdw
smdw'
```

which includes the newline. With the word boundary in the regex, it makes sure we _only_ get `mdw`.

**Comment by oliverralbertini** [comment]
> are there any other scripts that use the master hostname in a similar way?

I will check on that.

**Pull request merged**